### ISSUE TITLE
No maximum length on passwords

### ISSUE THREAD

**Issue by jojonium** [C7C8/SoftEng-API-Site#24]
**Describe the bug**
The UI says that passwords can be at most 2048 characters, but it doesn't actually stop you from typing in more than this. For example I was able to successfully register an account with the full text of 'The Most Dangerous Game' as my password (approx. 57,000 characters). Unfortunately when I tried with the full text of 'Moby Dick' (approx. 1.2 million characters) it failed with a 500 error.

**To Reproduce**
1. Go to `https://apisite.crmyers.dev/login`
2. Type in an email
3. Type (or paste) a lot of text into the password field
4. Click register

**Expected behavior**
The password field should stop accepting text after 2048 characters (i.e. put `maxLength="2048"` in the `<input type="password">` element)

**Comment by C7C8** [comment]
Yes, you broke the bcrypt password hashing algorithm. Congratulations.

**Comment by C7C8** [comment]
Hey, wait a minute... Joe, you're a team coach! Want me to venmo you a cent or something?
